scripts/generate_schema/football/generate_football_schema.py

```diff
@@ -75,6 +83,12 @@ def generate_json_schema(uri):
                     variables[key]['temporal_coverage'] = None
                 if key in entity_col:
                     variables[key]['name_entity'] = set()
+                if key in str_type_col:
+                    variables[key]['semantic_type'] = ["http://schema.org/Text"]
+                if key in int_type_col:
+                    variables[key]['semantic_type'] = ["http://schema.org/Integer"]
+                if key in date_type_col:
+                    variables[key]['semantic_type'] = ["https://metadata.datadrivendiscovery.org/types/Time"]
```

Comment:
Since flatten_map is created on line 64, the if i == 0 block can be factored out of this for loop. 

Also you can do 

```
for i, _ in enumerate(all_data):
```

instead of `for i in range(len(all_data))`, but in this case, since `i` is no longer used, we can just do `for d in all_data`

Reply:
Thank you for this